### ISSUE TITLE
What is our Mission?

### ISSUE THREAD

**Issue by iqbalabd** [PyConAPAC/docs#2]
Before we go on on defining requirements on PyCon APAC, I would like us to sit down and write down our Mission: Basically, why we are all here.

Like what PSF have on their mission page: https://www.python.org/psf/mission/

we should have something similar, although it doesn't have to be as detailed or as formal. Something like:
1. Expand python usage within the APAC region, through an annual PyCon APAC conference
2. Connect the APAC organizers to international sponsors via the PSF
3. Endorse PyCon APAC organizers

And make these public. Through these mission statements, it will help us by:
1. Having us all on the same page on where we want to head
2. So that people will understand what we're trying to do, and hopefully attract newcomers to help (as we all will not be here forever)
3. Gives us a point of reference when we make decisions that affects PyCon APAC

**Comment by yungyuc** [comment]
If what we want to achieve through PyCon APAC is the same as what PSF put in the mission statements, how about just follow it?  All we need to augment is the focus on APAC region.

**Comment by iqbalabd** [comment]
I feel with our current members and resources, letter by letter of what PSF is aiming for might not be what we can (or even _should_) be aiming for. For example, we can't distribute licenses or maintain intellectual property (because we are not an entity ourselves). We can't raise funds ourselves either, since we're not an entity too. And so on.

I think the keyword to what we can do (and actually is doing) is **to facilitate**, and our mission statement should be around that.

**Comment by yungyuc** [comment]
What I am thinking is to treat ourselves as part of PSF.  If we agree with the value stated in PSF missions, we are compatible to it.  In this way, we can treat PyCon APAC as a SIG under the PSF umbrella.  The [PyCon APAC WG](https://mail.python.org/mailman/listinfo/pyconapac) can be our entity in this setting.

**Comment by iqbalabd** [comment]
Ok, that sounds like a good direction to head. I do know we did talk about this during the last APAC with Van, but I as understand there was little headway. I think it is best to have a more concrete understanding on what we want our purpose is to be vs. what PSF would like us to do, if we want to move under the PSF umbrella, which means this discussion requires PSF input, before we go ahead and set guidelines (and enforce them) for PyCon APAC.

**Comment by vanzaj** [comment]
IMO, it is better to discuss this on pyconapac@python.org. Not everybody is on github.

**Comment by yungyuc** [comment]
@vanzaj agree.  This general discussion should go to the PyCon APAC WG.

An interesting thing is that we now have
1. pycon-organizers-apac@googlegroups.com
2. pyconapac@python.org (PyCon APAC WG)
3. https://github.com/PyConAPAC/docs/

It's clear that discussion against the guideline/cfh should be here in github.  It's easier to make revision using pull request.  But it's unclear to me how to distinguish the pycon-organizers-apac and pyconapac lists.  They are both public, while the former has a stricter rule of membership.

**Comment by vanzaj** [comment]
@yungyuc, yes, discussions about specific points of the cph guidelines can happen here. I meant more general discussions about "Mission" and relationship with PSF.

**Comment by iqbalabd** [comment]
What do we do about this then?

pycon-organizers-apac@googlegroups.com has a larger audience than pyconapac@python.org I think, which makes it much more suitable to discuss overarching matters like **mission statements** because I think we should have as much input and make it an open discussion as possible.

**Comment by bengkeatliew** [comment]
Hi all

I'm sorry for coming in late with the discussion. Its getting more and more difficult for me to engage more actively with my work commitments. 

1) Personally, I have been encouraged by the enthusiasm shown. I am excited with Iqbal taking the initiative and putting up the proposal on behalf of MY with hosting PyCon APAC 2017 even though we have raised questions on the proposal document.

2) I think 'authority or responsibility' was mentioned earlier. I recognize that we are really a disparate group, brought together only by our enthusiasm to share our common interest. We don't have any legal authority (as of now and not sure if we will) and neither do I have the same on the brand that is "PyCon APAC". I am just thankful that I had the opportunity to share a "dream" then and some of you were prepared to pick up on it. There are many gaps to our "structure" and in many ways, we basically have good people working on good intent on goodwill. 

3) Because of the above, if the proposal document sounds rather restrictive because _requirements_ were indicated, which seems to suggest that we have some kind of authority, I think we can agree to change the wording. Iqbal is correct... they are guidelines but I would like to think guidelines that offer the wisdom of past organizers. And shld be considered seriously nonetheless. So I am happy for any of us to re-look the wording/content of the document again.

4) And even as we do, are there others interested in hosting PyCon APAC in 2017 outside of MY? Let us hear from you too! 

5) On the PSF... I am not sure how we can progress with them really. We raised some matters with them in the past 2 years but momentum always gets lost. It just seems that we are really too "far". Iqbal.. your trip to the US might help to connect us. I did the same when I went to Chicago in 2009 and it helped kickstart things in SG and the idea of APAC. Your trip might help us do the same. 

6) Mission Statement? Yup... I would agree with Iqbal though I would fine-tune some of the statements a bit e.g.

A. Provide a platform for Python users from the APAC region to connect and collaborate through an annual PyCon APAC conference 
(not sure if we can directly influence the original 'Expand python usage' with just an annual conference. I think its really down to the respective in-country user groups, etc. if its to really happen) 

B. Support PyCon APAC organizers by
- Funding
- Connecting organizers to the International Sponsorship program via the PSF (still a to do)
- Have a rotating point of contact every year for the local organizers to talk to if they have questions
